### PR TITLE
Make metamorphism configurable, fix and update authoring page

### PR DESCRIPTION
**Pull request by pjanik**

### CODE REVIEW

File: js/components/authoring.tsx [hunk 1]
```diff
@@ -43,19 +45,16 @@ const VIEW_OPTIONS: Option[] = [
 const SKIPPED_OPTIONS: Option[] = ["authoring", "planetWizard", "planetWizardSteps", "sidebar", "preset", "modelId", "densityWordInPlanetWizard"];
 
 // All the options manually defined in various sections.
-const CUSTOM_OPTIONS: Option[] = [];
-CUSTOM_OPTIONS.concat(MAIN_OPTIONS, VIEW_OPTIONS, SKIPPED_OPTIONS).map(opt => typeof opt === "string" ? opt : opt[0]);
+const CUSTOM_OPTIONS: Option[] = [].concat(MAIN_OPTIONS as any, VIEW_OPTIONS as any, SKIPPED_OPTIONS as any)
```
Comment (kswenson) [inline js/components/authoring.tsx:48]: Would this work as a means of eliminating the `any`s?
```suggestion
const CUSTOM_OPTIONS: Option[] = [...MAIN_OPTIONS, ...VIEW_OPTIONS, ...SKIPPED_OPTIONS]
```

File: js/components/sidebar-menu.tsx [hunk 2]
```diff
@@ -52,13 +52,15 @@ export default class SidebarMenu extends BaseComponent<IProps, IState> {
   toggleVelocities: any;
   toggleVolcanicEruptions: any;
   toggleWireframe: any;
+  toggleMetamorphism: any;
```
Comment (kswenson) [inline js/components/sidebar-menu.tsx:55]: Not that it matters much, but isn't the type here just `() => void` for the `toggle...` functions and then `(value: any) => void` for the `change...` functions?

**Review by kswenson — APPROVED** [review]
👍 Looks good -- just a couple comments on types

File: js/components/sidebar-menu.tsx (hunk 2 above)
Comment (pjanik) [reply js/components/sidebar-menu.tsx]: Thanks, I've added these types.

File: js/components/authoring.tsx (hunk 1 above)
Comment (pjanik) [reply js/components/authoring.tsx]: It helped actually, thanks. :) Weirdly, even without the `.map()` call TS wasn't complaining about types in this case, but it's a nicer approach anyway.

**Comment by pjanik** [comment]
I've addressed all the comments, so merging this PR. Thanks!

**Pull request merged**